Esmorga-Backend/esmorga-backend
# [MOB-132] BE - POST Login phase 2

## Pull Request Checklist

- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-132)**

## Testing Guide
Add `LOGIN_ATTEMPTS_TTL` and `MAX_LOGIN_ATTEMPTS` to `.env`.
Try login with an unverified user and an active user 5 times to see if it is blocked.
```
curl --location 'http://localhost:3000/v1/account/login' \
--header 'Content-Type: application/json' \
--data-raw '{
    "email": "login-block@yopmail.com",
    "password": "SuperSecret1"
}'
```
